docs/user-guide/commands/argocd_app_create.md

```diff
@@ -27,7 +27,7 @@ argocd app create APPNAME [flags]
 	argocd app create nginx-ingress --repo https://kubernetes-charts.storage.googleapis.com --helm-chart nginx-ingress --revision 1.24.3 --dest-namespace default --dest-server https://kubernetes.default.svc
 
 	# Create a Kustomize app
-	argocd app create kustomize-guestbook --repo https://github.com/argoproj/argocd-example-apps.git --path kustomize-guestbook --dest-namespace default --dest-server https://kubernetes.default.svc --kustomize-image gcr.io/heptio-images/ks-guestbook-demo=0.1
+	argocd app create kustomize-guestbook --repo https://github.com/argoproj/argocd-example-apps.git --path kustomize-guestbook --dest-namespace default --dest-server https://kubernetes.default.svc --kustomize-image gcr.io/heptio-images/ks-guestbook-demo:0.1
```

Comment:
Was that a manual change, or was it created by `codegen`?

Reply:
This was a manual change the `=` is not a valid separator for an image tag, so I corrected it to `:`

Reply:
Ah, that explains - this part of the docs is auto-generated from the CLI code since recently. So I think you should make this change in the CLI [here](https://github.com/argoproj/argo-cd/blob/6ddd98c4f8998f0ebaad83b9b338e5debad52ac3/cmd/argocd/commands/app.go#L128) and run `codegen` again.

Reply:
ah okay, will update. one more commit ;)

Reply:
sorry I messed up the rebase, I need to regenerate once more.